packages/backend/src/common/tool/index.ts

```diff
@@ -1,9 +1,7 @@
 export const randomString = (length = 10): string => {
   let result = '';
   while (result.length < length) {
-    result += Math.random()
-      .toString(36)
-      .substr(2, 10);
+    result += Math.random().toString(36).substr(2, 10);
```

Comment:
run 'yarn build' in backend, prettier makes the previous change, there are a lot of conflicts because of it.

Reply:
rename `.prettierrc.json` to `.prettierrc`, and vscode will use it, instead overriding with default rules